### PR TITLE
Upgrade HyPhy tools to 2.5.4

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

### CODE REVIEW

File: tools/hyphy/macros.xml [hunk 1]
```diff
@@ -68,11 +76,27 @@
     </xml>
     <xml name="citations">
         <citations>
-            <citation type="doi">10.1093/bioinformatics/bti079</citation>
+         <citation type="bibtex">
```
Comment (bgruening) [inline tools/hyphy/macros.xml]: why are we not using the DOI here as well?

Comment (davebx) [reply tools/hyphy/macros.xml]: From @spond on slack:

> If you pull the doi citation, there are some weird character issues
> Hyphens don't render correctly
> Which is why I put the explicit citation in place

Comment (mvdbeek) [reply tools/hyphy/macros.xml]: We fixed that very recently in 20.01, can you check it out again? doi is much preferable

**Comment by bgruening** [comment]
> NameMapper.NotFound: cannot find 'mcmc'

One error that I say.

**Pull request merged**